packages/cli/src/requests.ts

```diff
@@ -392,7 +392,7 @@ export type BinaryDataRequest = AuthenticatedRequest<
 // ----------------------------------
 
 export declare namespace SSO {
-	export type AcsRequest = AuthenticatedRequest<
+	export type AcsRequest = AuthlessRequest<
```

Comment:
I'd like to keep all the SAML types lumped together, tbh., see packages/cli/src/sso/saml/types/requests.ts